content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded) for the server to become available
```

Comment:
Move the part about the deadline out of a parenthetical. Just make it another sentence, that "the deadline still applies."

Reply:
Done.